components/error_code/src/raft.rs

```diff
@@ -31,8 +30,8 @@ impl ErrorCodeExt for Error {
             Error::CodecError(_) => CODEC_ERROR,
             Error::Exists { .. } => EXISTS,
             Error::NotExists { .. } => NOT_EXISTS,
-            Error::RequestSnapshotDropped => REQUEST_SNAPSHOT_DROPPED,
             Error::ConfChangeError(_) => CONF_CHANGE_ERROR,
+            _ => unreachable!(),
```

Comment:
I suggest to write the type explicitly to avoid accidentally missing new definitions in the future.